### PR TITLE
Math Magicians React Project - Single Page App

### PR DESCRIPTION
**Pull request by jmontanholi**
## In this feature branch I implemented the following: 

* #### Home page
* #### Quote page
* #### Calculator page
   * #### Calculator component  
* #### Nav component
   * #### Routers to the pages

### CODE REVIEW

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
Hello @jmontanholi 😃, 

Great work and effort you put into this project so far.

## Highlights 🚀 

There are some excellent points that you have made perfectly. Keep up! 💪

- Routes are used very well
- Great separation of the CSS modules through the pages
- Well done using import for the CSS 

However, there is always room for improvement.

> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. 😄

 ## Required Changes ♻️

- [ ] While building the app, I faced this error (look at the image below) Kindly fix it 😄 🙏 
Screenshot

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [x] It'd be much nicer for the README file if you attached an image of the app to it 😄 

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by jmontanholi** [comment]
> Hello @jmontanholi 😃,
> 
> Great work and effort you put into this project so far.
> 
> ## Highlights 🚀
> There are some excellent points that you have made perfectly. Keep up! 💪
> 
> * Routes are used very well
> * Great separation of the CSS modules through the pages
> * Well done using import for the CSS
> 
> However, there is always room for improvement.
> 
> > Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. 😄
> 
> ## Required Changes ♻️
> * [x]  While building the app, I faced this error (look at the image below) Kindly fix it 😄 🙏
>   Screenshot
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> * [x]  It'd be much nicer for the README file if you attached an image of the app to it 😄
> 
> Cheers and Happy coding!👏👨‍💻💻
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Hello! I simply cannot find this error, this path does not exist in the files anymore. ".\src\components\calculator.js" is replaced everywhere with the right name, could you please try to pull changes to see if this is the error? This path is nowhere inside my files.

File: src/pages/CalculatorPage.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+/* eslint-disable import/extensions */
+/* eslint-disable react/no-unescaped-entities */
+import React from 'react';
+import Calculator from '../components/Calculator.js';
```
Comment (Davidosky007) [inline src/pages/CalculatorPage.js]: * Like the previous reviewer pointed out the error still persist 
image
* Hint 💭 `../components/Calculator.js` calculator.js should be lowercase and not uppercase

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
## Hi @jmontanholi 👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/pages/CalculatorPage.js (hunk 1 above)
Comment (jmontanholi) [reply src/pages/CalculatorPage.js]: image
This is the file in my folder. I don't know why it keeps happening this issue to code reviewers.

Comment (jmontanholi) [reply src/pages/CalculatorPage.js]: I can't reproduce that on my end.

**Comment by Davidosky007** [comment]
## Hi @jmontanholi 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 
job-welldone-gif

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**